internal/ethapi/sequencer.go

```diff
@@ -0,0 +1,56 @@
+package ethapi
+
+import (
+	"context"
+	"fmt"
+
+	client "github.com/astriaorg/go-sequencer-client"
+	sqproto "github.com/astriaorg/go-sequencer-client/proto"
+	"github.com/ethereum/go-ethereum/log"
+)
+
+const rollupChainID = "ethereum"
+
+func sendTransactionToSequencer(ctx context.Context, txBytes []byte) error {
+	signer, err := client.GenerateSigner()
+	if err != nil {
+		return err
+	}
+
+	// default tendermint RPC endpoint
+	c, err := client.NewClient("http://localhost:26657")
```

Comment:
Same here.

Reply:
done, I just used env vars, let me know if that works